src/test/assembly/x86_64-floating-point-clamp.rs

```diff
@@ -0,0 +1,27 @@
+// Floating-point clamp is designed to be implementable as max+min,
+// so check to make sure that's what it's actually emitting.
+
+fn main() {}
+
+// assembly-output: emit-asm
+// compile-flags: -O -C llvm-args=-x86-asm-syntax=intel
```

Comment:
Apparently I should have tried my suggested test out locally; I made lots of mistakes 😬

If you remove `fn main` and change 
```suggestion
// compile-flags: --crate-type=lib -O -C llvm-args=-x86-asm-syntax=intel
```

it'll pass.

<details>

At least this file passed locally for me:

```rust
// Floating-point clamp is designed to be implementable as max+min,
// so check to make sure that's what it's actually emitting.

// assembly-output: emit-asm
// compile-flags: --crate-type=lib -O -C llvm-args=-x86-asm-syntax=intel
// only-x86_64

// CHECK-LABEL: clamp_demo:
#[no_mangle]
pub fn clamp_demo(a: f32, x: f32, y: f32) -> f32 {
    // CHECK: maxss
    // CHECK: minss
    a.clamp(x, y)
}

// CHECK-LABEL: clamp12_demo:
#[no_mangle]
pub fn clamp12_demo(a: f32) -> f32 {
    // CHECK-NEXT: movss   xmm1
    // CHECK-NEXT: maxss   xmm1, xmm0o
    // CHECK-NEXT: movss   xmm0
    // CHECK-NEXT: minss   xmm0, xmm1
    // CHECK-NEXT: ret
    a.clamp(1.0, 2.0)
}
```

</details>